docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
 
       # create default database and user on starting
       CLICKHOUSE_DEFAULT_ACCESS_MANAGEMENT: 1
-      CLICKHOUSE_DB: scratchdb
+      CLICKHOUSE_DB: scratch
```

Comment:
Should this config file also include clickhouse-keeper?

Reply:
@poundifdef No. Keeper is currently not enabled. I will have it in a different PR.

Let me know if you would rather have it combined here.

Reply:
I think it makes sense to combine it here. Otherwise the code will not work when someone pulls it down to run locally (ON CLUSTER will fail without keeper configured.)